packages/modules/web_themes/koala/source/src/components/DailyTotals.vue

```diff
@@ -0,0 +1,408 @@
+      >
+        <div v-if="currentPowerVisible">Aktuelle Leistung</div>
+        <div>Tageswerte</div>
+      </div>
+      <q-table
+        flat
+        :rows="rows"
+        :columns="columns"
+        row-key="id"
+        hide-header
+        hide-bottom
+        dense
+        separator="none"
+        class="banner-table theme-text"
+      >
+        <template #body="props">
+          <q-tr
+            :props="props"
+            :style="{ '--row-bg': props.row.backgroundColor }"
+          >
+            <q-td key="icon" :props="props">
+              <img
+                :src="props.row.icon"
+                :alt="props.row.title"
+                class="icon"
+                :style="{ width: iconSize + 'px', height: iconSize + 'px' }"
+              />
+            </q-td>
+
+            <q-td
+              v-if="componentNameVisible"
+              key="title"
+              :props="props"
+              class="text-body2 text-weight-bold"
+            >
+              {{ props.row.title }}
+            </q-td>
+
+            <q-td
+              v-if="socValueVisible"
+              key="soc"
+              :props="props"
+              class="text-right"
+            >
+              <span v-if="props.row.id === 'battery'"
+                >{{ props.row.soc }}%</span
+              >
+            </q-td>
+
+            <q-td v-if="currentPowerVisible" key="arrow" :props="props">
+              <q-icon
+                :name="
+                  props.row.id === 'house'
+                    ? 'horizontal_rule'
+                    : arrowDirection(props.row.id).noCurrent
+                      ? 'horizontal_rule'
+                      : 'double_arrow'
+                "
+                :class="{
+                  'rotate-180': arrowDirection(props.row.id).rotate180,
+                }"
+              />
+            </q-td>
+
+            <q-td
+              v-if="currentPowerVisible"
+              key="power"
+              :props="props"
+              class="text-right text-body2"
+            >
+              {{ String(props.row.power).replace('-', '') }}
+            </q-td>
+
+            <!-- Gap column stretches -->
+            <q-td key="gap" :props="props"></q-td>
+
+            <q-td
+              key="rightLabel"
+              :props="props"
+              class="text-right text-weight-bold"
+            >
+              <template
+                v-if="['battery', 'chargepoint'].includes(props.row.id)"
+              >
+                <div>Geladen:</div>
+                <div>Entladen:</div>
+              </template>
+              <template v-else-if="props.row.id === 'grid'">
+                <div>Bezug:</div>
+                <div>Einspeisung:</div>
+              </template>
+              <template v-else-if="props.row.id === 'pv'">
+                <div>Ertrag:</div>
+              </template>
+              <template v-else-if="props.row.id === 'house'">
+                <div>Energie:</div>
```

Comment:
"Energie" kann sowohl Import als auch Export sein.
```suggestion
                <div>Verbrauch:</div>
```